site/src/components/sidebar.js

```diff
@@ -1,6 +1,8 @@
 // sidebar component
 const html = require('choo/html')
+const raw = require('choo/html/raw')
 const css = require('sheetify')
+const mrk = require('../util/mrk')
```

Comment:
Watch out - util/mrk supports channelrefs and stuff!

Reply:
Not sure how to fix this without a pretty big update to mrk - e.g., temporarily disabling/enabling features for one particular `mrk` call, or setting "categories" (such as "permit `# headings` but not channel refs in the "outside of messages" category). Not *too* big of a deal, though, but still something to do later!

Reply:
Categories can be done with `has(token, beforeToken)`